product_docs/docs/pgd/5/cli/command_ref/commit-scope/create.mdx

```diff
@@ -0,0 +1,43 @@
+---
+title: pgd commit-scope create
+navTitle: create
+---
+
+## Synopsis
+
+The `pgd commit-scope create` command is used to create a commit scope in the EDB Postgres Distributed cluster.
+
+## Syntax
+
+```shell
+pgd commit-scope create <COMMIT_SCOPE> [OPTIONS] <RULE_DEFINITION> [GROUP_NAME]
+```
+
+Where `<COMMIT_SCOPE>` is the name of the commit scope to create.
+
+The `<RULE_DEFINITION>` is the rule that defines the commit scope. The rule specifies the conditions that must be met for a transaction to be considered committed. See [Commit Scopes](/pgd/latest/commit-scopes) and [Commit Scope Rules](/pgd/latest/commit-scopes/commit-scope-rules/) for more information on the rule syntax.
+
+The optional `[GROUP_NAME]` is the name of the group to which the commit scope belongs. If omitted, it defaults to the top-level group.
+
+## Options
+
+No command specific options. See [Global Options](/pgd/latest/cli/command_ref/#global-options).
+
+## Examples
+
+### Creating a Commit Scope
+
+The following example creates a commit scope named `abc1` with the rule `ANY 2 (dc1) on replicated group commit`:
+
+```shell
+pgd commit-scope create abc1 "ANY 2 (dc1) on replicated group commit"
```

Comment:
can we use a better commit-scope name that is more standard and more user-friendly